pages/_/components.tsx

```diff
@@ -0,0 +1,123 @@
+import { Button, ButtonLink } from 'components/core/Button/Button';
+import DeprecatedButton from 'components/core/Button/DeprecatedButton';
+import styled from 'styled-components';
+
+export default function DesignPage() {
```

Comment:
this is an incredible idea! does putting things under `/_/` tell nextjs that this won't be available in prod? i guess it's not a huge deal if it's public

Reply:
Nah, it's just security by obscurity. I couldn't think of a name so I went with `_` haha. We could easily add rules in the Next config or something to keep this from being visible in prod, but given that this is all open source, I don't think it's a bad thing that this would exist on the web for reals.

Reply:
love it! great idea to use `_`